erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -1441,7 +1425,7 @@ def set_asset_status(self, asset):
 			asset.set_status("Sold" if self.docstatus == 1 else None)
 
 	def make_loyalty_point_redemption_gle(self, gl_entries):
-		if cint(self.redeem_loyalty_points):
+		if cint(self.redeem_loyalty_points and self.loyalty_points and not self.is_consolidated):
```

Comment:
This can break `					dr_or_cr = gle.debit - gle.credit` in `account/utils.py` as this would generate a data structure of `int - NonType` or `NoneType - int` if `self.loayalty_points` is `None`.